frontend/src/libs/components/input/input.tsx

```diff
@@ -5,20 +5,24 @@ import {
   type FieldValues,
 } from 'react-hook-form';
 
+import { Icon } from '~/libs/components/components.js';
 import { getValidClassNames } from '~/libs/helpers/helpers.js';
 import { useFormController } from '~/libs/hooks/hooks.js';
+import { type IconType } from '~/libs/types/types.js';
 
 import styles from './styles.module.scss';
 
 type Properties<T extends FieldValues> = {
   control: Control<T, null>;
   errors: FieldErrors<T>;
-  label: string;
+  label?: string;
```

Comment:
```suggestion
  label: string;
```
use `isLabelVisuallyHidden` if you need to hide a label